pkg/controllers/cstorvolumeconfig/controller_base.go

```diff
@@ -324,3 +331,28 @@ func (c *CVCController) processNextWorkItem() bool {
 
 	return true
 }
+
+func isV1PodDisruptionBudgetSupported(client discovery.DiscoveryInterface) bool {
+	const (
+		podDisruptionBudgetKind      = "PodDisruptionBudget"
+		podDisruptionBudgetStableApi = "policy/v1"
+	)
+
+	//Checking if the policy/v1 API contains the CronJob Kind
+	apiList, err := client.ServerResourcesForGroupVersion(podDisruptionBudgetStableApi)
```

Comment:
I guess this uses the discovery API, can we not pass a `metav1.ListOptions{}` to only get a specific kind of resource, this way we wouldn't have to iterate on it, from what I can recall it doesn't but please check if you've got time.

Reply:
No, I didn't find an interface to query if a specific object is present.